src/components/Form/index.js

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useEffect } from 'react'
+import axios from 'axios';
+
+function UserInput (props) {
+
+  const [name, setName] = useState('')
+  const [GitData, setGitData] = useState('')
+  //let str = JSON.stringify(GitData)
+
+  
+  const handleSubmit = event => {
+    event.preventDefault()
+
+    async function getGitData() {
+      const response = await axios.get(`https://api.github.com/users/${name}/repos`)
+
+      // create a for loop to iterate over each repo and retrieve data //
+      
+      const gitData = response.data 
+      //console.log(gitData)
+      setGitData(gitData)
+      //console.log(GitData)
+      const result = []
```

Comment:
You fetched all the correct data, however it is not being rendered onto the page. Instead of a for loop like you've done above, you could probably map out the response data (which you have assigned to gitData) and map through a basic html structure 

For example:

```
const renderData = gitData.map((response => (
<h2> response.name </h2>
<p>response.forks_count</p>
 
etc.